<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(119, 118, 123);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="link1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>link1</string>
    </property>
   </widget>
   <widget class="QLabel" name="link2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>link2</string>
    </property>
   </widget>
   <widget class="QLabel" name="link3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>link3</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="link1SpinBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>25</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="link2SpinBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>55</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="link3SpinBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>85</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ArmButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Arm</string>
    </property>
   </widget>
   <widget class="QLabel" name="joint1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>joint1</string>
    </property>
   </widget>
   <widget class="QLabel" name="joint2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>joint2</string>
    </property>
   </widget>
   <widget class="QLabel" name="joint3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>joint3</string>
    </property>
   </widget>
   <widget class="QSlider" name="joint1Slider">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="joint2Slider">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="joint3Slider">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="joint1Angle">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>148</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="joint2Angle">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>178</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="joint3Angle">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>208</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RestButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
